obal/__init__.py

```diff
@@ -47,19 +117,75 @@ def takes_package_parameter(self):
 
         return any('packages' in play['hosts'] for play in plays)
 
+    @property
+    def playbook_variables(self):
+        """
+        The playbook variables that should be exposed to the user
+
+        This is extracted from the metadata.
+        """
+        return self.metadata['variables']
+
+    @property
+    def help_text(self):
+        """
+        The help text if available. This is the first line from the help in the metadata.
+        """
+        return self.metadata['help'].split('\n', 1)[0] if self.metadata['help'] else None
```

Comment:
maybe `.splitlines()` instead of `.split('\n')`?

Reply:
I thought that only splitting it once was more efficient but there's probably little difference

Reply:
Yeah. I would not think there is much diff. But also would not force you to change :)